articles/key-vault/key-vault-ovw-storage-keys.md

```diff
@@ -11,15 +11,16 @@ manager: barbkess
 ms.date: 03/01/2019
 # Customer intent: As a developer I want storage credentials and SAS tokens to be managed securely by Azure Key Vault.
 ---
+
 # Azure Key Vault managed storage account - CLI
 
-> [!NOTE]
-> [Azure storage integration with Azure Active Directory (Azure AD) is now in preview](https://docs.microsoft.com/azure/storage/common/storage-auth-aad). We recommend using Azure AD for authentication and authorization, which provides OAuth2 token-based access to Azure storage, just like Azure Key Vault. This allows you to:
-> - Authenticate your client application using an application or user identity, instead of storage account credentials. 
+> [!NOTE][azure storage integration with azure active directory (azure ad) is now in preview](https://docs.microsoft.com/azure/storage/common/storage-auth-aad). We recommend using Azure AD for authentication and authorization, which provides OAuth2 token-based access to Azure storage, just like Azure Key Vault. This allows you to:
```

Comment:
Can you change it to Azure Storage in the first few words and Azure AD

Reply:
Updated all instances of `azure storage` and `azure ad` to `Azure Storage` and `Azure AD`.